values.yaml

```diff
@@ -13,8 +13,6 @@
 # limitations under the License.
 #
 
-# Use Palisade-services values.yaml mostly, but set some useful defaults here
-
```

Comment:
Possibly useful to keep this in, demos that this values.yaml is just overrides on the services one

Reply:
This was not something that was removed by me but will add it back in.